sdks/python/container/base_image_requirements.txt

```diff
@@ -25,6 +25,7 @@
 
 avro==1.8.2;python_version<="2.7"
 avro-python3==1.8.2;python_version>="3.4"
+collapsing-thread-pool-executor>="2018.6"
```

Comment:
We pin versions in this file.

Reply:
I see, will revert

Reply:
Removed the change to this file.

Reply:
It is fair to change this to 
`collapsing-thread-pool-executor=="2018.6"`

Removing is also fine.